### ISSUE TITLE
Design Intent - Accessibility Feedback - Cartographers, MHV on VA.gov, MHV Secondary Navigation

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#71910]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- It sounds like there's not a ton of enthusiasm around this from Matt, but just for completeness I'll mention the [minimal header](https://design.va.gov/components/header/header-minimal) component as something to explore in the context of Option 1. Definitely some major tradeoffs if you go that route and I'm not really convinced it solves a ton of problems either, but it does address the "more than one menu" problem.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Coding note:** I think the best way to code this will be something like:
   ```
   <nav aria-labelledby="some-id">
      <h2 id="some-id">My HealtheVet</h2>
      <ul>
         <li><a href="...">Link 1</a></li>
         <li><a href="...">Link 2</a></li>
         ...
   </nav>
   ``` 
   What I'm not sure about (and what I would definitely recommend exploring with @AngelaFowler82!) is whether the `My HealtheVet` should be a heading, or if it would be better as a non-semantic element like a `span`. I'm leaning toward H2 but I defer to Angela's experience here.
- **Consider:** Visible text is often the cleanest path to an accessible experience, which is why Option 2 appeals to me. There might be design considerations that force some/all nav links into something collapsed on some set of viewports. But I'd definitely encourage exploring designs that keep the navigation visible as much as possible.
- **Consider:** Martha raised a good point about the use of chevrons to indicate, well, a lot of things on VA.gov. For viewports where the secondary menu is collapsed, I think it would be worth testing text + chevron against text + hamburger icon (or some other menu-y icon?) to see if there's one that performs better.
- **Consider:** Looking at Option 3, my biggest concern would be whether the My HealtheVet trigger for opening/closing the collapsed menu would be clearly identifiable as a `button`. The normal rules of link/button visual design kind of fall apart in headers and footers, so this ultimately comes down to whether users are able to correctly identify what the element does in testing. But with that in mind, I think it's worth exploring visual features that make it a little more button-y. Not a must but something to consider.

**Comment by sterkenburgsara** [comment]
@briandeconinck thank you so much for all of this feedback! We're with you on the benefit of visible text and it sounds like there is a lot of alignment on preference for the 2nd and 3rd designs for testing, so I look forward to learning how these do with users. I have a clarifying question about the nudge to code this as a <nav> element with an H2. In my mind, this would be placed visually above the page H1. Are you saying it's okay to have an H2 above an H1? Can you describe in a different way what you mean if I have this wrong? // cc @AngelaFowler82 

I have another question about the new breadcrumb breakpoint, which is now 480 px (down from 768 px) for USWDS v3. I lean toward a preference for aligning navigation experiences, so I'd be inclined to take on the same breakpoint for a horizontal nav like options 2 or 3. Are there a11y concerns with that at all that we should be aware of?

**Comment by wesrowe** [comment]
@briandeconinck @AngelaFowler82, just wanted to surface @sterkenburgsara's question above as we're currently coding prototypes for research. The question being: Brian's code sample above shows an H2 inside a component (`<nav>`) that will ultimately sit above the page's H1 in the DOM. This seems to break page hierarchy. Is that okay?
Cc @jzucadi

**Comment by wesrowe** [comment]
Closing post-staging review.
